.gitlab-ci.yml

```diff
@@ -284,15 +300,23 @@ doc:refman:
   dependencies:
     - build:base
 
+doc:refman:dune:
+  <<: *dune-ci-template
+  variables:
+    <<: *dune-ci-template-vars
+    DUNE_TARGET: refman-html
+  artifacts:
+    <<: *dune-ci-template-artifacts
+    paths:
+      - _build/default/doc/sphinx_build/html
+
 doc:ml-api:odoc:
-  stage: test
-  dependencies:
-    - build:egde:dune:dev
-  script: make -f Makefile.dune apidoc
+  <<: *dune-ci-template
   variables:
-    OPAM_SWITCH: edge
+    <<: *dune-ci-template-vars
+    DUNE_TARGET: apidoc
   artifacts:
-    name: "$CI_JOB_NAME"
+    <<: *dune-ci-template-artifacts
     paths:
       - _build/default/_doc/
```

Comment:
NB: this change means this artifact now has an expire key. (maybe make a note in the commit message)

Reply:
Ok. I do prefer to expire by default, specially in ephemeral builds; but of course this could be overriden if our deployment strategy changes.

Reply:
Maybe one month could be a more sensible outcome here tho.